docs/secure-connections/sauce-connect/advanced/specifications.md

```diff
@@ -0,0 +1,61 @@
+---
+id: specifications
+title: Sauce Connect Proxy Specifications
+sidebar_label: Specifications
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+If you're a security or network administrator, you may find it useful to review the Sauce Connect Proxy specifications to better evaluate its security.
+
+## What You'll Need
+- Have a working understanding of [Sauce Connect Proxy architecture](/secure-connections/sauce-connect/advanced/architecture).
```

Comment:
Some experience with Sauce Connect Proxy [recommended practices](/secure-connections/sauce-connect/proxy-tunnels).

Not sure how to phrase it - you know how to use Sauce Connect Proxy already.